### PR TITLE
feat: optimize compute vertex state

### PR DESCRIPTION
**Pull request by trungnotchung**

### CODE REVIEW

**Review by JanLewDev — APPROVED** [review]
lgtm

File: packages/object/src/index.ts [hunk 1]
```diff
@@ -196,22 +199,13 @@ export class DRPObject implements IDRPObject {
...
-		const state = Object.create(
-			Object.getPrototypeOf(fetchedState),
-			Object.getOwnPropertyDescriptors(structuredClone(fetchedState)),
-		).state;
+		const state = structuredClone(fetchedState);
 
-		for (const [key, value] of state.entries()) {
+		for (const [key, value] of state.state) {
 			drp[key] = value;
 		}
```
Comment (d-roak) [inline packages/object/src/index.ts:206]: there's no need for this when deps === 1

Comment (d-roak) [reply packages/object/src/index.ts:206]: nor lines 197-202

**Review by d-roak — APPROVED** [review]
lgtm

**Pull request merged**